package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fontaweswimm",
   "main": "src/fonts/fontaweswimm/style.css",
-  "version": "1.1.24",
```

Comment:
@asafchn as a rule of thumb, when you revert, the version number is never downgraded